.idea/compiler.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No need of this

Reply:
yeah...i am just adding a button that will allow the user to choose the above languages...will that do?

Reply:
yeah, you can add a preference screen for settings that will provide you a feature called switch preferences to switch between various languages. For implementing this you can search what a preference setting means in android :)

Reply:
yeah I tried using the preference setting screen...but it opens the entire list of languages available on android....but the app supports only 4 languages....so i made a button on top right ...such that the user can switch between the available languages....would you plz look at it once ...attaching the screenshots

Reply:
![image](https://user-images.githubusercontent.com/74768430/101602065-83891e80-3a23-11eb-8310-a9cb60520918.png)
![image](https://user-images.githubusercontent.com/74768430/101602111-97348500-3a23-11eb-86f2-371362d6a0ce.png)
![image](https://user-images.githubusercontent.com/74768430/101602172-b206f980-3a23-11eb-9be8-368d2f3d6e94.png)

Reply:
Added a new commit...plz look at it once :)

Reply:
Oops!! Forgot to translate the choose language text

Reply:
Now i think it should work fine :)

Reply:
> No need of this

Should I remove that line😊